db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_02_19_162054) do
+ActiveRecord::Schema.define(version: 2021_02_19_164926) do
```

Comment:
why this change is here?
you have not correctly rebased I think

Reply:
Probably.